doc/articles/features/using-il-linker-webassembly.md

```diff
@@ -1,4 +1,26 @@
-# Using the IL Linker for WebAssembly
+---
+uid: articles.features.illinker
+---
+
+# Using the IL Linker
+
+### Support for features
+
+In order to improve the size of the application, some platforms are providing the ability to unconditionally disable features if the app is known to not use them.
+
+|Feature|MSBuild property|Description|
+|-----|----|---|
+|External Drag and Drop|`DragDropExternalSupport`|Uno Platform supports drag and drop, and the support for dropping content from outside the app can be disabled using this property. Drag and Drop inside the application is still available when disabled.|
```

Comment:
DragDropExternalSupport => UnoDragDropExternalSupport

Reply:
Enable/Disable drag and dropping content from **outside** the app using this property. Drag and drop **inside** the app is still available when disabled.